.github/workflows/publish.yml

```diff
@@ -17,9 +17,9 @@ jobs:
       - uses: actions/cache@v1
         with:
           path: legacy/vendor/bundle
-          key: ${{ runner.os }}-gems-${{ hashFiles('**/Gemfile.lock') }}
+          key: ${{ runner.os }}-gems-v2-${{ hashFiles('**/Gemfile.lock') }}
```

Comment:
Can you add this in a separate PR

Reply:
Done @IsaacMurtagh. Can you please review the PR again?